CHANGES.md

```diff
@@ -100,6 +101,8 @@
 * Support for X source added (Java/Python) ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
 * Add streaming support to SnowflakeIO in Java SDK ([BEAM-9896](https://issues.apache.org/jira/browse/BEAM-9896  ))
 * Support reading and writing to Google Healthcare DICOM APIs in Python SDK ([BEAM-10601](https://issues.apache.org/jira/browse/BEAM-10601))
+* Add dispositions for SnowflakeIO.write ([BEAM-10343](https://issues.apache.org/jira/browse/BEAM-10343))
+* Add cross-language support to SnowflakeIO.Read now available in the Python module `apache_beam.io.external.snowflake` ([BEAM-9897](https://issues.apache.org/jira/browse/BEAM-9897)).
```

Comment:
apache_beam.io.snowflake ?

Reply:
Unfortunately Read was merged in apache_beam.io.external.snowflake
It's been moved to apache_beam.io.snowflake along with Write, that's why there is a need for backwards compatibility. I added the deprecation info to 2.25 notes.